core/ui/vault/settings/delete/index.tsx

```
@@ -0,0 +1,135 @@
+            description={vault.publicKeys.eddsa}
+          />
+        </List>
+        <Divider text={t('terms')} />
+        <List>
+          {terms.map((term, index) => (
+            <ListItem
+              extra={<StyledCheck value={termsAccepted[index]} />}
+              onClick={() => toggleCheckbox(index)}
+              title={t(term)}
+              hoverable
+            />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)